ui/urls.py

```diff
@@ -49,22 +49,22 @@
     ),
     url(
         r"^privacy-and-cookies/$",
-        core.views.PrivacyCookiesDomestic.as_view(),
+        core.views.PrivacyCookiesDomesticCMS.as_view(),
         name='privacy-and-cookies'
     ),
     url(
         r"^terms-and-conditions/$",
-        core.views.TermsConditionDomesticViewNegotiator.as_view(),
+        core.views.TermsConditionsDomesticCMS.as_view(),
```

Comment:
no feature flag?

Reply:
no need for one - with the first page added it was important to be able to roll back quickly, but now we can just keep replacing.